sparql/isDefinedBy.sparql

```diff
@@ -9,7 +9,7 @@ WHERE {
 
 FILTER isIRI(?s) .
 FILTER STRSTARTS(STR(?s), "http://purl.obolibrary.org/obo/") .
-BIND (LCASE(STRBEFORE(STRAFTER(STR(?s), "http://purl.obolibrary.org/obo/"), "_")) AS ?ont) . # use last instance of "_"
+BIND (LCASE(REGEX(STRAFTER(STR(?s), "http://purl.obolibrary.org/obo/"), "_[^_]+$")) AS ?ont) . 
```

Comment:
I think you're going to need `REPLACE` instead of `REGEX` here, to replace the matched part with an empty string and get back the ontology prefix.

Reply:
Good—I made one edit to fix some parentheses.